Phonebook.Frontend/.gitignore

```diff
@@ -1,47 +1,47 @@
-# See http://help.github.com/ignore-files/ for more about ignoring files.
-
-# compiled output
-dist/
-/tmp
-/out-tsc
-/src/changelog.md
-
-# dependencies
-node_modules/
-
-# IDEs and editors
-/.idea
-.project
-.classpath
-.c9/
-*.launch
-.settings/
-*.sublime-workspace
-
-# IDE - VSCode
-.vscode/*
-!.vscode/*.code-snippets
-!.vscode/settings.json
-!.vscode/tasks.json
-!.vscode/launch.json
-!.vscode/extensions.json
-
-# misc
-/.sass-cache
-/connect.lock
-/coverage
-/libpeerconnection.log
-npm-debug.log
-testem.log
-/typings
-yarn-error.log
-key.pem
-cert.pem
-
-# e2e
-/e2e/*.js
-/e2e/*.map
-
-# System Files
-.DS_Store
+# See http://help.github.com/ignore-files/ for more about ignoring files.
```

Comment:
don't commit stuff like that.

Reply:
i dont understand why, but i had to commit the .gitignore without any changes. 

VsCode and powershell show that the file is modified. 

Any idea why git does this ?

Reply:
line endings

Reply:
hopefully fixed

Reply:
thats the last time, after this it should be fixed